packaging/debian/postinst

```diff
@@ -25,34 +25,92 @@ DAEMON_USER=storj-storagenode
 
 . /usr/share/debconf/confmodule
 
+arch=$(dpkg --print-architecture)
+if [ "$arch" = "armhf" ]; then
+    arch="arm" # use GOARCH notation
+fi
+
+get_binary() {
+    binary=$1
+    tmpfile=$(mktemp /tmp/${binary}.zip.XXXXXX)
```

Comment:
I'm trying to think of a situation when `$TMPDIR` would be set to not `/tmp` or any other situation when `/tmp` wouldn't be writable. I'd like `/tmp/` removed from this line but it's more of a nitpick.

Reply:
@brimstone if you okay with this PR being merged could you please remove your change request?